charts/licensify/values.yaml

```diff
@@ -18,43 +18,49 @@ clamav:
     type: ClusterIP
     port: 3310
 
-licensifyAdmin:
-  name: licensify-admin
-  replicaCount: 1
-  port: 9950
-  healthcheckPath: "/"
-  ingress:
-    annotations: {}
-  service:
-    port: 80
-
-licensifyBackend:
-  name: licensify-backend
-  replicaCount: 1
-  port: 9920
-  healthcheckPath: "/"
-  service:
+apps:
+  licensifyAdmin:
+    name: licensify-admin
+    replicaCount: 1
+    port: 9950
+    healthcheckPath: "/"
+    ingress:
+      enabled: true
+      annotations: {}
+    service:
+      port: 80
+
+  licensifyBackedn:
```

Comment:
```suggestion
  licensifyBackend:
```

Reply:
Ah thank you! I couldn't workout why the diff was so weird